deployment_scripts/linode-marketplace-harbor/harbor-deploy.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+set -e
+trap "cleanup $? $LINENO" EXIT
+
+##Linode/SSH security settings
+#<UDF name="user_name" label="The limited sudo user to be created for the Linode" default="">
+#<UDF name="password" label="The password for the limited sudo user" example="an0th3r_s3cure_p4ssw0rd" default="">
+#<UDF name="disable_root" label="Disable root access over SSH?" oneOf="Yes,No" default="No">
+#<UDF name="pubkey" label="The SSH Public Key that will be used to access the Linode (Recommended)" default="">
+
+## Domain Settings
+#<UDF name="token_password" label="Your Linode API token. This is needed to create your WordPress server's DNS records" default="">
+#<UDF name="subdomain" label="Subdomain" example="The subdomain for the DNS record: www (Requires Domain)" default="">
+#<UDF name="domain" label="Domain" example="The domain for the DNS record: example.com (Requires API token)" default="">
+
+## harbor Settings 
+#<UDF name="harbor_admin_password" label="The Harbor admin password">
+#<UDF name="harbor_db_password" label="The Harbor database password ">
+#<UDF name="soa_email_address" label="Admin Email for the Harbor server and Let's Encrypt SSL certificate">
+
+# git repo
+export GIT_REPO="https://github.com/jcotoBan/marketplace-apps.git"
```

Comment:
needs to be updated to https://github.com/akamai-compute-marketplace/marketplace-apps.git"

Reply:
Done